### ISSUE TITLE
100% indexing never ends

### ISSUE THREAD

**Issue by blockagency-co** [OllieJones/fast-woo-order-lookup#15]
Does this plugin keep generating the index over and over again? Because I get this message constantly on the site:

“Fast Woo Order Lookup indexing still in progress. 100% complete. See the Scheduled Actions status page for details.”

And when this happens, the site backend and order lookup slows down again.

PS: At the site where I installed it, we receive new orders every minute.

**Comment by OllieJones** [comment]
Thanks for the report. I'm investigating this.

**Comment by OllieJones** [comment]
Yeah, truly a defect, thanks again! I use an option to keep track of how far along the indexing process is.  I was updating that option incorrectly for orders that arrive after the indexing process is complete.

**Comment by blockagency-co** [comment]
Hey Ollie, thanks to you for creating such an awesome solution for order searching on Woo!
